### ISSUE TITLE
Make `getFile` available in DSL scripts

### ISSUE THREAD

**Issue by toorog** [structurizr/dsl#332]
### Description

All properties of `StructurizrDslPluginContext` are available within a `!script`.
I need to access the [`getFile()` property](https://github.com/structurizr/dsl/blob/57f036f5f7bb7e94725d2f0be89dff957623e022/src/main/java/com/structurizr/dsl/StructurizrDslPluginContext.java#L36C17-L36C27) as well, because my script relies on files that are located in the same directory as the `workspace.dsl`. Otherwise there is no chance for the script to know the directory the DSL file is located in.

TLDR: Request is to make the workspace file property of  `StructurizrDslPluginContext` available to scripts

### Priority

Low

### Resolution

I have no budget, please add this feature for free

**Comment by simonbrowndotje** [comment]
Once released, you'll be able to do `context.getDslFile()`.
